package.json

```diff
@@ -7,7 +7,7 @@
     "================================================================================"
   ],
   "name": "@aurodesignsystem/auro-dropdown",
-  "version": "2.10.1",
+  "version": "2.10.2",
```

Comment:
no need to update this manually because Auro use semantic versioning.

Reply:
oh, okay, thanks @fajar-apri-alaska, i will revert this.